src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    public Figure getRandomFigure(int maxSize) {
+        Random rand = new Random();
+        String color = ColorSupplier.getRandomColor();
+        int randFigureIndex = rand.nextInt(AvailableFigure.values().length);
```

Comment:
You should define a constant for the number of available figures instead of calling `values().length` directly in the `nextInt` method.